pontoon/settings/base.py

```diff
@@ -97,6 +97,85 @@ def _default_from_email():
 # Google Cloud Translation API key
 GOOGLE_TRANSLATE_API_KEY = os.environ.get("GOOGLE_TRANSLATE_API_KEY", "")
 
+# Pontoon locale codes supported by Google Cloud AutoML Translation Project ID
+#
+# Source:
+# https://cloud.google.com/translate/automl/docs/languages#supported_codes_for_language_variants
+GOOGLE_AUTOML_SUPPORTED_LOCALES = [
```

Comment:
Why is this here and not in the DB?

Reply:
Would you rather see something like a `Locale.google_automl_enabled` boolean?

Reply:
No strong opinion, but thought there could be a reason that I wasn't seeing right away. Fine either way.